.concourse/pipeline.yaml.gomplate

```diff
@@ -418,8 +413,7 @@ jobs:
       image_resource:
         type: registry-image
         source:
-          repository: thulioassis/bazel-docker-image
-          tag: 2.0.0
+          repository: splatform/catapult
```

Comment:
Same as above: pin to version please.  It might be useful to use YAML anchors here.

Reply:
Done